lib/ret/media_search.ex

```diff
@@ -343,11 +343,15 @@ defmodule Ret.MediaSearch do
   def available?(:tenor), do: has_resolver_config?(:tenor_api_key)
   def available?(:twitch), do: has_resolver_config?(:twitch_client_id)
 
-  defp sketchfab_search(query) do
+  def sketchfab_search(query) do
```

Comment:
I'm a noobie when it comes to testing, so I appreciate the feedback. I might not have thought to test sketchfab downloads specifically, except that they happen to have regressed while I was working on this feature. (See https://github.com/mozilla/reticulum/pull/483 ). 

I'm open to suggestions on how to write better tests. If testing the sketchfab api specifically were something we wanted to do, would you recommend that I move this function into a separate module so that it would have to be public in order to be used in this file (`media_search.ex`)? (That way it wouldn't be public _only_ for the tests, but this feels like cheating... I feel like I'm missing the point.)

Reply:
Yeah, that would feel like cheating. I'd say we should think about ways of testing this code without having to expose implementation-specific functions. For example, there could be an integration test that tests the public interface that we already have for sketchfab search, and ideally we would mock out the actual call to the sketchfab API, because our goal here is to test our code surrounding the sketchfab API, and not the sketchfab API itself.